mne/minimum_norm/resolution_metrics.py

```diff
@@ -0,0 +1,397 @@
+# -*- coding: utf-8 -*-
+# Authors: Olaf Hauk <olaf.hauk@mrc-cbu.cam.ac.uk>
+#
+# License: BSD (3-clause)
+"""Compute resolution metrics from resolution matrix.
+
+Resolution metrics: localisation error, spatial extent, relative amplitude.
+Metrics can be computed for point-spread and cross-talk functions (PSFs/CTFs).
+"""
+import numpy as np
+
+from mne import SourceEstimate
+from mne.utils import logger, verbose
+
+
+@verbose
+def resolution_metrics(resmat, src, function, kind, metric):
+    """Compute spatial resolution metrics for linear solvers.
+
+    Parameters
+    ----------
+    resmat : array, shape (n_orient*n_locations, n_locations)
+        The resolution matrix.
+        If not a square matrix and if the number of rows is a multiple of
+        number of columns (e.g. free or loose orientations), then the Euclidean
+        length per source location is computed (e.g. if inverse operator with
+        free orientations was applied to foward solution with fixed
+        orientations).
+    src : instance of SourceSpaces
+        Source space object from forward or inverse operator.
+    function: str
+        'psf'|'ctf'
+        Whether to compute metrics for columns (point-spread functions, PSFs)
+        or rows (cross-talk functions, CTFs) of the resolution matrix.
+    kind : str
+        What kind of resolution feature to consider.
+        Options are 'localization_error' | 'spatial_extent' | 'amplitude'.
+    metric : str
+        The exact resolution metric for the kind of resolution feature.
+        This must match with 'kind'. The allowed options are:
+        kind='localization_error':
+            'peak': Peak localization error (PLE), Euclidean distance between
+                    peak
+                    and true source location.
+            'cog': Centre-of-gravity localisation error (CoG), Euclidean
+                   distance between CoG and true source location.
+        kind='spatial_extent':
+            'sd': spatial deviation (e.g. Molins et al.).
```

Comment:
add the ref at the bottom in a section:

```
References
-----------
```

it should be after the Returns section. See how it's done elsewhere